Content/.template.config/template.json

```diff
@@ -22,17 +22,13 @@
             "dataType": "choice",
             "defaultValue": "saturn",
             "choices": [
-                {
-                    "choice": "suave",
-                    "description": "lightweight, non-blocking web server (standalone)"
-                },
                 {
                     "choice": "giraffe",
-                    "description": "a functional micro web framework based on ASP.NET Core"
+                    "description": "A native functional ASP.NET Core web framework for F# developers"
                 },
                 {
                     "choice": "saturn",
-                    "description": "web development framework which implements the server-side MVC pattern"
+                    "description": "A modern web framework that focuses on developer productivity, performance, and maintainability"

```

Comment:
If we're amending that already, how about mentioning that Saturn also uses ASP.NET Core? Otherwise it *might* scare newcomers

Reply:
I just took the comment from the Saturn homepage, but happy to change it - @Krzysztof-Cieslak any thoughts?